packages/nx/src/plugins/js/lock-file/lock-file.ts

```diff
@@ -182,7 +185,7 @@ function getLockFilePath(packageManager: PackageManager): string {
     return NPM_LOCK_PATH;
   }
   if (packageManager === 'bun') {
-    return BUN_LOCK_PATH;
+    return BUN_LOCK_PATH; // defaulting to binary lockfile
```

Comment:
I feel like we should be checking the Bun version and if `>= 1.2` this should be defaulting to the text based one?